geoh5py/data/reference_value_map.py

```diff
@@ -63,7 +63,9 @@ def validate_value_map(cls, value_map: np.ndarray | dict) -> np.ndarray:
 
         if isinstance(value_map, np.ndarray) and value_map.dtype.names is None:
             if value_map.ndim == 1:
-                value_map = {i: str(val) for i, val in enumerate(set(value_map))}
+                unique_set = set(value_map)
+                unique_set.discard(0)
```

Comment:
why discard 0?
I guess that's the bug to correct because that's the only change. 

a label can't be named "0" in GA?

Reply:
0 is reserved for "Unknown"